BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -328,76 +362,133 @@ def dump (self):
 			print (self.hex_str (component.data, 'data') + ' %s' % str(component.data[:4]))
 
 	def create (self, layout):
-		container_sig, container_file, alignment, auth_type, key_file, region_size = layout[0]
-		if type(alignment) is str:
-			if alignment == '':
-				alignment = '0x1000'
-			alignment = int (alignment, 0)
+
+		# for monolithic signing, need to add a reserved _SG_ entry to hold the auth info
+		mono_sig = '_SG_'
+		is_mono_signing = True if layout[-1][0] == mono_sig else False
+
+		# get the first entry in layout as CONTAINER_HDR
+		container_sig, container_file, image_type, auth_type, key_file, alignment, region_size = layout[0]
+
+		if alignment == 0:
```

Comment:
Just in case, if alignment is not necessary, what's the value in alignment? 0 goes to 0x1000 all the time.

Reply:
alignment should be set to 1 if the alignment is not required.